tv_apps/smart-system/style/browser_context_menu/browser_context_menu.css

```diff
@@ -0,0 +1,9 @@
+form[role="dialog"][data-type="action"].contextmenu {
+  transform: translateY(0);
```

Comment:
Oh, I see. The original animation should be removed. 
Seems `transition: none` alone works well so maybe we can remove the line above?